test/test_helper.exs

```diff
@@ -34,6 +34,8 @@ defmodule AbsinthePlug.Test.Schema do
         {:ok, get_book_by_isbn(args.isbn)}
       end)
     end
+
+    field :books, list_of(:book)
```

Comment:
I am probably missing something, but this doesn't seem to be used

Reply:
I added this to make sure that query fields not explicitly selected aren't showing up, but it could probably benefit from an explicit `refute` in the tests to both clarify and explicitly test.